lib/smart_answer_flows/marriage-abroad/outcomes/_synonyms_of_cp.govspeak.erb

```diff
@@ -114,7 +114,7 @@
   - ’domestic partnership’ (in Nova Scotia)
   - ’union civile’, or ’civil union’ (in Quebec)
 <% when 'new-zealand' %>
-  You may be able to get married or enter into a ‘civil union’ at the British embassy or consulate in New Zealand. Civil unions are recognised as civil partnerships under British law.
+  Civil unions are recognised as civil partnerships under British law.
```

Comment:
We do tend to get this info from FCO, just didn't know it helped to include it in the commit note. We can start putting reasons into the gdoc for devs if it's useful.